src/exchanges.js

```diff
@@ -31,6 +31,11 @@ let commonRoutingKey = function(options) {
                         'characters aside from - and _ have been stripped.',
       maxSize:          100,
       required:         true,
+    }, {
+      name:             'installationID',
+      summary:          'The installation which had an event.',
```

Comment:
As I note below, I don't think this is necessary -- we can leave the routing keys as they are.